src/app/auth/auth.component.ts

```diff
@@ -1,13 +1,15 @@
-import {Component, OnDestroy, OnInit} from '@angular/core';
-import {AuthService, AuthState} from '../core/services/auth.service';
-import {Subscription} from 'rxjs';
-import {FormBuilder, FormGroup, NgForm, Validators} from '@angular/forms';
-import {HttpErrorResponse} from '@angular/common/http';
-import {ErrorHandlingService} from '../core/services/error-handling.service';
-import {Router} from '@angular/router';
-import {GithubService} from '../core/services/github.service';
-import {PhaseService} from '../core/services/phase.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { AuthService, AuthState } from '../core/services/auth.service';
+import { Subscription } from 'rxjs';
+import { FormBuilder, FormGroup, NgForm, Validators } from '@angular/forms';
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorHandlingService } from '../core/services/error-handling.service';
+import { Router } from '@angular/router';
+import { GithubService } from '../core/services/github.service';
+import { PhaseService } from '../core/services/phase.service';
```

Comment:
In future, avoid doing unrelated 'housekeeping' changes in a PR. A PR should contain only things related to the intended fix so that it can be reverted later, if needed.

Reply:
Sure 👍